src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -139,28 +139,31 @@
                                                     </div>
                                                 </div>
                                             </div>
-                                            {% if message.htmlBody is defined %}
+                                            {% if message.htmlBody() is defined %}
                                                 {# Email instance #}
-                                                <div class="tab">
-                                                    <h3 class="tab-title">HTML preview</h3>
-                                                    <div class="tab-content">
-                                                        <pre class="prewrap" style="max-height: 600px">
-                                                            <iframe
-                                                                src="data:text/html;base64;charset=utf-8,{{ collector.base64Encode(message.htmlBody()) }}"
-                                                                style="height: 80vh;width: 100%;"
-                                                            >
-                                                            </iframe>
-                                                        </pre>
+                                                {% set htmlBody = message.htmlBody() %}
+                                                {% if htmlBody is not null %}
```

Comment:
This fixes https://github.com/symfony/symfony/issues/44703

Reply:
This will hide the preview tab for emails without html content. We should consequently either hide the HTML tab as well, or show an empty preview tab.

Reply:
indeed, I will make them conditional as well